src/Dotnet.Script.Core/Templates/program.publish.template

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Scripting.Hosting;
+using Microsoft.CodeAnalysis.Scripting.Hosting;
+using System.Threading.Tasks;
+using static System.Console;
+
+namespace dotnetPublishCode
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                var globals = new CommandLineScriptGlobals(Console.Out, CSharpObjectFormatter.Instance);
+                foreach (var arg in args)
+                    globals.Args.Add(arg);
+
+                var factoryMethod = typeof(scriptAssembly).GetMethod("<Factory>");
+                if (factoryMethod == null) throw new Exception("couldn't find factory method to initiate script");
+
+                // todo: not sure what the second parameter is, using 'null' for now
+                var invokeTask = factoryMethod.Invoke(null, new object[] { new object[] { globals, null } }) as Task<int>;
+                var invokeResult = invokeTask.Result;
+                if (invokeResult != 0) WritePrettyError($"Error result: '{invokeResult}'");
+            }
+            // todo: not getting the full script stack trace
```

Comment:
what do you mean?

Reply:
so i noticed that when i would throw an error in the script, I wouldn't get any of the script stack trace information to print out.  I can see if that was because I wasn't using your emitter with debug information maybe?

Reply:
for example, if i threw an error on line 25 of the main.csx file, when writing out the error it wouldn't give me any indication that the error actually occurred on 25, or if there was previous stack trace data

Reply:
this is probably because you emit without symbols - if you compare the emitter code I mentioned before, it emits symbols too and line numbers are available in the symbols only